### PR TITLE
Objenesis bootstrapping agreement, 2024-05-06

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

File: agreements/bootstrapping/2024-05-06-bootstrapping-objenesis.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+status: draft
```
Comment (ebullient) [inline agreements/bootstrapping/2024-05-06-bootstrapping-objenesis.md:2]: Oversight on my part. 🙄 Can you remove this 3 line draft indicator?

**Review by ebullient — CHANGES_REQUESTED** [review]

File: agreements/bootstrapping/2024-05-06-bootstrapping-objenesis.md (hunk 1 above)
Comment (henri-tremblay) [reply agreements/bootstrapping/2024-05-06-bootstrapping-objenesis.md]: Sure. Done

**Review by ebullient — APPROVED** [review]

**Review by cealsair — APPROVED** [review]

**Review by maxandersen — APPROVED** [review]

**Review by aalmiray — APPROVED** [review]

**Review by gavinking — APPROVED** [review]

**Comment by kenfinnigan** [comment]
@henri-tremblay could you please rebase this PR, and we can then get this merged. Thanks

**Comment by henri-tremblay** [comment]
Sure. Done.

**Comment by kenfinnigan** [comment]
Welcome to Commonhaus!

**Pull request merged**